function test(){
    try{
  
    }catch(err) {
        return 0;
    }finally{
        return 0;
    }
}